client/my-sites/site-indicator/site-indicator.jsx

```diff
@@ -294,8 +364,38 @@ export default React.createClass( {
 							</Animate>
 						</button>
 					</div>
-					: null }
+				}
+			</div>
+		);
+	}
+
+	render() {
+		const { site, siteIsJetpack } = this.props;
+
+		return (
+			<div className="site-indicator__wrapper">
+				{ siteIsJetpack && <QuerySiteConnectionStatus siteId={ site.ID } /> }
+				{ siteIsJetpack && <QuerySiteUpdates siteId={ site.ID } /> }
```

Comment:
:+1: indeed, removed in https://github.com/Automattic/wp-calypso/pull/13221/commits/6f85e99de8c902a23e3c3f7de63bfe2a249b695e.